packages/components/pages/f-account-info/src/components/AccountInfo.vue

```diff
@@ -366,7 +367,23 @@ export default {
                 await this.loadConsumerDetails({ api: this.consumerApi, authToken: this.authToken });
                 this.$log.info('Consumer details fetched successfully', standardLogTags);
             } catch (error) {
-                this.$log.error('Error fetching consumer details', error, standardLogTags);
+                // Debug - Temp logging - Check auth token state when GET fails
+                let endpointAuthTokenExpired = 0;
+                try {
+                    if (this.authToken) {
+                        const { exp } = jwtDecode(this.authToken);
+                        if (Date.now() >= exp * 1000) {
+                            endpointAuthTokenExpired = 1;
+                        }
+                    } else {
+                        endpointAuthTokenExpired = 1;
+                    }
+                } catch {
+                    // noop
+                    endpointAuthTokenExpired = -1;
+                }
```

Comment:
Can we put the `1`, `0` and `-1` into an object describing them instead of using magic numbers?